src/fsdocs-tool/BuildCommand.fs

```diff
@@ -186,6 +187,8 @@ type internal DocContent
 
               let isMd = inputFileFullPath.EndsWith(".md", true, CultureInfo.InvariantCulture)
 
+              let isPynb = inputFileFullPath.EndsWith(".ipynb", true, CultureInfo.InvariantCulture)
```

Comment:
```suggestion
              let isPynb = inputFileFullPath.EndsWith(".ipynb", StringComparison.Ordinal)
```

You actually want to compare without any culture at all.
https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#recommendations-for-string-usage

Reply:
Changed all 3 to StringComparison.OrdinalIgnoreCase. The old version ignored case so that `file.md` and `file.MD` are both treated as markdown files.